app/mailers/notifier.rb

```diff
@@ -76,6 +77,15 @@ def order_detail_status_change(order_detail, old_status, new_status, to)
 
   private
 
+  def attach_ical(order)
+    order.order_details.map(&:reservation).compact.each do |reservation|
+      calendar = ReservationCalendar.new(reservation)
+      attachments[calendar.filename] = {
```

Comment:
Have you been able to test this to see if gmail picks it up? We can always test from the open stage if you haven't.